extension/.gitignore

```diff
@@ -0,0 +1 @@
+/out
```

Comment:
super nit: add a new line at the end. I see this error while reviewing:

<img width="310" alt="image" src="https://user-images.githubusercontent.com/57374379/193907359-1b42283d-6381-4f3c-9146-29b9ad436703.png">

Reply:
love a good super nit! updated

Reply:
🤷updated, but I still see the warning in github